Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2018"
 
 [dependencies]
 rand = "0.8"
+rand_xoshiro = "0.6"
```

Comment:
this sounds like some weeb shit

Reply:
It's not it's from italy
https://prng.di.unimi.it/

Reply:
oh ok 👍